### PR TITLE
CFIN-404 Config of latest tuning results and metadata weighting and curations

### PR DESCRIPTION
**Pull request by ncn1**
There are curation entries for 19 out of 59 keywords/phrases tested to push the single subject standard Bachelors degree to the top of the results where it proved necessary.

It turns out manual metadata weighting additions are not overwritten when the results of tuning are applied to a collection, so the padre-opts file contains '-wmeta.courseTitle=1.000 -wmeta.courseDepartment=1.0000f ' which helped to greatly reduce the number of curation entries.

Possibly simplest to review the changes by using the Funnelback UI for the curations, and trying out searches using https://york.funnelback.co.uk/search?collection=york-uni-courses&profile=_default_preview&smeta_contentType=course&num_ranks=35&query=<search term>.

The kill_exact.cfg file was removed from the collection once the change to use smeta_contentType=course parameter in the Courses API went live, but had not been removed from this repo.

### CODE REVIEW

File: config/curator.json [hunk 1]
```diff
@@ -245,256 +188,225 @@
...
   "trigger" : {
     "type" : "And",
     "triggers" : [ {
       "type" : "Or",
       "triggers" : [ {
-        "type" : "ExactQuery",
-        "triggerQuery" : "Sociology",
-        "ignoreCase" : true
+        "type" : "AllQueryWords",
+        "triggerWords" : [ "theoretical" ]
```
Comment (FergusMcGlynn) [inline config/curator.json:404]: The trigger word "theoretical" seems a bit generic for targeting "theoretical physics". Could we make the trigger "theoretical physics"? Do we need curation for the query "theoretical physics"?

Comment (ncn1) [reply config/curator.json:404]: Good spot - must have got left in while I was playing about with this - initially, Physics came at the top, as it is also a curated keyword, so I had to add to the curation of 'Physics' a trigger of 'and keyword does not match theoretical'.

File: config/curator.json [hunk 2]
```diff
@@ -245,256 +188,225 @@
...
...https://www.york.ac.uk/study/undergraduate/courses/bsc-archaeology-biology-environment/", "https://www.york.ac.uk/study/undergraduate/courses/msci-archaeology-biology-environment/", "https://www.york.ac.uk/study/undergraduate/courses/bsc-natural-science-biology-chemistry-physics/", "https://www.york.ac.uk/study/undergraduate/courses/msci-natural-science-biology-chemistry-physics/", "https://www.york.ac.uk/study/undergraduate/courses/bsc-biophysical-science/", "https://www.york.ac.uk/study/undergraduate/courses/msci-biophysical-science/", "https://www.york.ac.uk/study/undergraduate/courses/bsc-natural-sciences-chemistry-maths-physics/", "https://www.york.ac.uk/study/undergraduate/courses/msci-natural-sciences-chemistry-maths-physics/", "https://www.york.ac.uk/study/undergraduate/courses/bsc-mathematical-bioscience/", "https://www.york.ac.uk/study/undergraduate/courses/msci-mathematical-bioscience/", "https://www.york.ac.uk/study/undergraduate/courses/bsc-nanoscience/", "https://www.york.ac.uk/study/undergraduate/courses/msci-nanoscience/", "https://www.york.ac.uk/study/undergraduate/courses/bsc-neuroscience/", "https://www.york.ac.uk/study/undergraduate/courses/msci-neuroscience/" ]
```
Comment (FergusMcGlynn) [inline config/curator.json:275]: This seems like quite a lot of URLs to promote. I appreciate that natural sciences is a bit of a special case. But do we need this much curation?

File: config/curator.json [hunk 3]
```diff
@@ -245,256 +188,225 @@
...
+    }, {
       "type" : "Or",
       "triggers" : [ {
-        "type" : "ExactQuery",
-        "triggerQuery" : "Philosophy",
-        "ignoreCase" : true
+        "type" : "NotAny",
+        "triggers" : [ {
+          "type" : "AllQueryWords",
+          "triggerWords" : [ "theoretical" ]
```
Comment (FergusMcGlynn) [inline config/curator.json:319]: Will we still need this if there is a separate rule for "theoretical physics"? There's a possibility that Funnelback is clever enough to infer this this sub-clause from another rule.

File: config/curator.json (hunk 1 above)
Comment (FergusMcGlynn) [reply config/curator.json]: Maybe if we make this one `"triggerWords" : [ "theoretical", "physics" ]` then Funnelback will be clever enough to prioritise this rule for "theoretical physics" over the "physics" rule? Might be worth a try.

File: config/curator.json (hunk 3 above)
Comment (ncn1) [reply config/curator.json:319]: It wasn't, only added this when became clear it put Physics at the top.

File: config/curator.json (hunk 2 above)
Comment (ncn1) [reply config/curator.json:275]: Some of the Natural Sciences degrees were appearing way down in the list, after lots of other non-natural science courses, so just seemed simplest to curate a list in alphabetical order.

Comment (FergusMcGlynn) [reply config/curator.json:275]: Might it be worth checking with Claire/Katie (maybe others?) to see whether alphabetical ordering is the right way to go?

**Review by FergusMcGlynn — APPROVED** [review]

**Review by RichardAllum — APPROVED** [review]

**Pull request merged**